### PR TITLE
#167220594: fix incorrect automation status in details modal

### PR DESCRIPTION
**Pull request by katunold**
#### What does this PR do?
- This PR fixes incorrect automation status displayed in details modal.

#### Description of Task to be completed?
**actual**
- The details modal displays the automation status as a success even when some of the automation was not successful. 
**expected**
- The details model is only expected to display the automation status as a success only if only all the automation was successful.

#### How should this be manually tested?
1. Clone the repo, checkout to the `bg-card-success-response-167220594` and install all the dependencies.
2. run the server using this command `yarn start`
3. head over to the browser and paste this url `http://localhost:3000/`
4. click on the info icon on any of the cards, a modal with the automation details should show up showing success on if all the automation displayed in details are all successful otherwise it should display false.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[167220594](https://www.pivotaltracker.com/story/show/167220594)

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
image

### CODE REVIEW

File: src/components/AutomationDetails/index.jsx [hunk 1]
```diff
@@ -28,10 +28,10 @@ class AutomationDetails extends PureComponent {
...
       formatDates, retryingAutomation, handleRetryAutomation, data,
     } = this.props;
-    const { slackAutomations, emailAutomations, nokoAutomations } = modalContent;
-    const slackStatus = (slackAutomations && slackAutomations.status) || '';
-    const emailStatus = (emailAutomations && emailAutomations.status) || '';
-    const nokoStatus = (nokoAutomations && nokoAutomations.slackActivities) || '';
+    const { slackAutomations, emailAutomations, freckleAutomations } = modalContent;
+    const slackStatus = !!(slackAutomations && slackAutomations.status === 'success');
+    const emailStatus = !!(emailAutomations && emailAutomations.status === 'success');
+    const freckleStatus = !!(freckleAutomations && freckleAutomations.slackActivities === 'success');
```
Comment (bozicschucky) [inline src/components/AutomationDetails/index.jsx]: hey @katunold , we changed the data from freckle to noko.
Please change the data to change match the new made changes.

File: src/components/AutomationDetails/index.jsx [hunk 2]
```diff
@@ -47,24 +47,15 @@ class AutomationDetails extends PureComponent {
           <div className="name-group">
             <h1 className="developer-name">{modalContent.fellowName}</h1>
             <div className="tooltip-container" id="ais-link-icon">
-              <i
-                className="fas fa-external-link-alt"
-                title={`Open ${modalContent.fellowName} AIS profile`}
-                onClick={() => window.open(`https://ais.andela.com/people/${modalContent.fellowId}`)
-                    }
-              />
+              <i className="fas fa-external-link-alt" title={`Open ${modalContent.fellowName} AIS profile`} onClick={() => window.open(`https://ais.andela.com/people/${modalContent.fellowId}`)} />
```
Comment (bozicschucky) [inline src/components/AutomationDetails/index.jsx]: hey @katunold , this line is too long which doesn't conform to the eslint rules.

File: src/components/AutomationDetails/index.jsx [hunk 3]
```diff
@@ -47,24 +47,15 @@ class AutomationDetails extends PureComponent {
...
           <h1 className="partner-name">{modalContent.partnerName}</h1>
           <div className="automation-action">
             <h1 className="automation-type">{modalContent.type}</h1>
             <h1 className="dot">.</h1>
-            {slackStatus === 'failure' || emailStatus === 'failure' || nokoStatus === 'failure' ? (
-              <h1 className="automation-status">Failed</h1>
-            ) : (
-              <h1 className="automation-status success">Success</h1>
-            )}
+            {slackStatus === false || emailStatus === false || freckleStatus === false ? <h1 className="automation-status">Failed</h1> : <h1 className="automation-status success">Success</h1>}
```
Comment (bozicschucky) [inline src/components/AutomationDetails/index.jsx]: the same applies to this line

File: yarn.lock [hunk 4]
```diff
@@ -3299,9 +3299,9 @@ cyclist@~0.2.2:
   resolved "https://registry.yarnpkg.com/cyclist/-/cyclist-0.2.2.tgz#1b33792e11e914a2fd6d6ed6447464444e5fa640"
 
 cypress@^3.3.1:
-  version "3.4.0"
-  resolved "https://registry.yarnpkg.com/cypress/-/cypress-3.4.0.tgz#8053ee107eb6309f26abd57e882d05578bdc3391"
-  integrity sha512-vUE+sK3l23fhs5qTN3dKpveyP0fGr37VmK3FSYaTEjbqC/qh4DbA1Ych/3bLStUpHP4rpE5KAx7i1s/tpdD9vQ==
+  version "3.3.2"
```
Comment (bozicschucky) [inline yarn.lock:3309]: why is the version of cypress downgraded from a higher version..?

**Review by bozicschucky — CHANGES_REQUESTED** [review]

File: src/components/AutomationDetails/index.jsx (hunk 1 above)
Comment (katunold) [reply src/components/AutomationDetails/index.jsx]: Thanks for pointing that out, let me fix it immediately @bozicschucky

File: yarn.lock (hunk 4 above)
Comment (katunold) [reply yarn.lock:3309]: I really don't know why

**Review by ssewilliam — APPROVED** [review]

**Pull request merged**